pkg/metrics/reporter.go

```diff
@@ -0,0 +1,23 @@
+package metrics
+
+import (
+	"context"
+
+	"go.opencensus.io/tag"
+)
+
+type Reporter struct {
+	Ctx context.Context
+}
+
+// NewMetricsReporter creaters a Reporter for generic metrics.
+func NewMetricsReporter() (*Reporter, error) {
+	ctx, err := tag.New(
```

Comment:
tag.New(context.Background()) is equivalent to just context.Background(). tag.New only has an effect if other arguments are passed to it.

Reply:
I actually no longer need this whole file, but I'll apply the same comment to other locations in my code where it's relevant.